cookbooks/bcpc-hadoop/recipes/graphite_disk_wrapper.rb

```diff
@@ -0,0 +1,18 @@
+# assign prepare a direcotry structure for graphite to install itself to
+reservation_requests = node[:bcpc][:hadoop][:disks][:reservation_requests]
+
+if reservation_requests.include?("graphite_disk") then
+  disk_index = reservation_requests.index("graphite_disk")
+
+  directory "/disk/#{disk_index}/graphite_disk" do
```

Comment:
Nit: Just out of curiosity, why the name `graphite_disk` instead of just `graphite`? With this code the effect will be `/opt/graphite -> /disk/0/graphite_disk`. Would it be better to have `/opt/graphite -> /disk/0/graphite`?